manifests/s/Splashtop/SplastopPersonal/3.46.9.7601/Splashtop.SplastopPersonal.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Splashtop.SplastopPersonal

```

Comment:
That looks like a spelling mistake in the package name... (missing 'h'?)

Reply:
fixed